src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -230,6 +257,19 @@ export class GlobalMap extends React.Component<Props, State> {
               trackUserLocation: true,
             })
           }}
+          onPress={async event => {
+            this.setState({
+              activeShowID: null,
+            })
+            const { screenPointX, screenPointY } = event.properties
```

Comment:
The idea here is we use user's tap x and y values and the map zoom level to fetch the cluster from our cluster engine at which we map the points back to the shows